src/Fable.Elmish.fsproj

```diff
@@ -18,6 +11,13 @@
     <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
     <PackageReference Include="FSharp.Core" Version="4.1.*" />
     <PackageReference Include="Fable.Core" Version="1.0.0-narumi-*" />
+    <PackageReference Include="Fable.PowerPack" Version="1.0.0-narumi-*" />
```

Comment:
I don't understand yet how `paket` interacts with the new `dotnet` tools, but if we just remove the `PackageReference`s, how would the compiler know to reference the Fable.PowerPack (for example)? Also, unless we switch to `paket` + `paket.template` and list the dependencies, how would we tell the consumer he needs that dependency?